styles/contato.css

```diff
@@ -8,6 +8,10 @@
 }
 
 @media screen and (min-width: 320px) and (max-width: 655px) {
```

Comment:
WDYT corrigir para:
`@media (min-width: 320px) and (max-width: 655px) {` ?
Assim que corrigido, precisa entender o pq o menu mobile ainda não funciona.